sale_commission/migrations/13.0.1.0.0/post-migration.py

```diff
@@ -42,3 +51,23 @@ def migrate(env, version):
     """
         ).format(sql.Identifier(openupgrade.get_legacy_name("invoice_line_id"))),
     )
+    openupgrade.logged_query(
+        env.cr,
+        """
+            UPDATE account_move am
+            SET commission_total = ai.commission_total
+            FROM account_invoice ai
+            WHERE ai.id = am.old_invoice_id
+                AND ai.commission_total IS NOT NULL
+                AND ai.commission_total != 0.0
+        """,
+    )
+    openupgrade.logged_query(
+        env.cr,
+        """
+            UPDATE account_move_line aml
+            SET commission_free = ail.commission_free
+            FROM account_invoice_line ail
+            WHERE aml.old_invoice_line_id = ail.id AND ail.commission_free
```

Comment:
I'm wondering if this field can contain nulls by default and then you need to assign all values